smartsim/_core/_cli/build.py

```diff
@@ -87,13 +94,19 @@ def __init__(self):
             logger.info("Checking requested versions...")
             self.versions = Versioner()
 
+            if self.keydb:
+                self.versions.REDIS = Version_("6.2.0")
+                self.versions.REDIS_URL = "https://github.com/EQ-Alpha/KeyDB"
+                self.versions.REDIS_BRANCH = "v6.2.0"
+                CONFIG.conf_path = Path(CONFIG.core_path, "config", "keydb.conf")
```

Comment:
probably good to check this path if we don't somewhere else.

Reply:
As in check if the path leads to a file that exists?